docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,286 @@
+
+Provide more context by answering these questions:
+
+* **Can you reproduce the problem in both debug and release mode?**
+* **Did the problem start happening recently** (e.g. after updating to
+  a new version/commit of SpECTRE) or was this always a problem?
+* If the problem started happening recently, **can you reproduce the
+  problem in an older version/commit?** What's the most recent
+  version/commit in which the problem doesn't happen?
+* **Can you reliably reproduce the issue?** If not, provide details
+  about how often the problem happens and under which conditions it
+  normally happens.
+* **Can you reproduce the problem on another machine?**
+
+Include details about your configuration and environment:
+
+* **Add as an attachment** (or add the contents of) the following:
+  - The text output by SpECTRE (including any stack trace)
+  - The input file(s)
+  - $SPECTRE_BUILD_DIR/LibraryVersions.txt
+* **What is the name and version of the OS you're using**?
+* If possible (for SXS computers or HPC systems), a **path to a run
+  directory** that is accessible by SpECTRE core developers.
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion
+for SpECTRE, including completely new features and minor improvements
+to existing functionality. Following these guidelines helps
+maintainers and the community understand your suggestion and find
+related suggestions.
+
+Before creating enhancement suggestions, please **perform a
+[search](https://github.com/sxs-collaboration/spectre/issues)** to see
+if the enhancement has already been suggested. If it has, add a
+comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub
+issues](https://guides.github.com/features/issues/).  When you are
+creating an enhancement suggestion, please **include as many details
+as possible** as you fill in the template.
+
+* **Use a clear and descriptive title** for the issue to identify the
+  suggestion.
+* **Provide a step-by-step description of the suggested enhancement**
+  in as many details as possible.
+* **Explain why this enhancement would be useful** to most SpECTRE users.
+
+### Your First Code Contribution
+
+Unsure where to begin contributing to SpECTRE? You can start by
+looking through these `good first issue` and `help wanted` issues:
+
+* [Good first issues][good-first-issue] - issues which should only
+  require a few lines of code, and a test or two.
+* [Help wanted issues][help-wanted] - issues which should be a bit
+  more involved than `good first issue`s.
+
+#### Local development
+
+SpECTRE can be developed locally. For instructions on how to do this,
+see the following sections in the [SpECTRE
+documentation](https://spectre-code.org/):
+
+* [Installing SpECTRE](installation.html)
+* [Running Status Checks Locally](travis_guide.html)
+
+### Pull Requests
```

Comment:
The first sentence of this section should be something along the lines of "Code contributions to SpECTRE follow a (pull request model)[https://help.github.com/articles/about-pull-requests/]".